### PR TITLE
map init

### PR DESCRIPTION
**Pull request by jusung1459**

### CODE REVIEW

File: serious_stuff/lib/main.dart [hunk 1]
```diff
@@ -3,6 +3,7 @@ import 'package:serious_stuff/screens/HomePage.dart';
 import 'package:serious_stuff/constants.dart' as Constants;
 
 void main() {
+
```
Comment (cl418) [inline serious_stuff/lib/main.dart:6]: Probably don't need the extra space here

File: serious_stuff/lib/screens/HomePage.dart [hunk 2]
```diff
@@ -1,5 +1,10 @@
+
```
Comment (cl418) [inline serious_stuff/lib/screens/HomePage.dart:1]: Extra space here too

File: serious_stuff/lib/screens/HomePage.dart [hunk 3]
```diff
@@ -21,6 +26,35 @@ class HomePage extends StatefulWidget {
...
+  @override
+  void initState() {
+    super.initState();
+    _checkLocationPermission();
+  }
+
+  void _checkLocationPermission() async {
+    _serviceEnabled = await location.serviceEnabled();
+    if(!_serviceEnabled) {
```
Comment (cl418) [inline serious_stuff/lib/screens/HomePage.dart:42]: I believe we're going with spaces after control structure words?
```suggestion
    if (!_serviceEnabled) {
```

File: serious_stuff/lib/screens/MapPage.dart [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+Completer<GoogleMapController> _controller = Completer();
+
+const LatLng _center = const LatLng(49.2780947,-122.9373926);
```
Comment (cl418) [inline serious_stuff/lib/screens/MapPage.dart:7]: Do we want to keep the coordinates as constants somewhere? If we don't need them again it probably doesn't matter much then

File: serious_stuff/lib/utils/Map.dart [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    super.initState();
+    _locationData = widget.location;
+  }
+
+  // set Markers on map
+  void _setMarkers(LatLng point) {
+    final String markerIdVal = 'marker_id_$_markerIdCounter';
+    _markerIdCounter++;
+    setState(() {
+      print(
```
Comment (cl418) [inline serious_stuff/lib/utils/Map.dart:49]: The print statement can probably just be one line

**Review by cl418 — APPROVED** [review]
Should be fine; there are just some minor style related things that could be changed

File: serious_stuff/lib/screens/MapPage.dart (hunk 4 above)
Comment (kachen10) [reply serious_stuff/lib/screens/MapPage.dart:7]: Also please have spaces after operators like `+`, `-` or `,`

File: serious_stuff/lib/utils/Map.dart [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+        Polygon(
+          polygonId: PolygonId(polygonIdVal),
+          points: polygonLatLngs,
+          strokeWidth: 2,
+          strokeColor: Colors.yellow,
+          fillColor: Colors.yellow.withOpacity(0.15),
+        )
+    );
+  }
+  @override
```
Comment (kachen10) [inline serious_stuff/lib/utils/Map.dart:90]: Please have a space between methods.

File: serious_stuff/pubspec.yaml [hunk 7]
```diff
@@ -29,11 +29,14 @@ dependencies:
...
   cupertino_icons: ^0.1.3
   flutter_speed_dial: ^1.2.5
+  google_maps_flutter: ^0.5.28+1
+  location: ^3.0.2
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
+
```
Comment (kachen10) [inline serious_stuff/pubspec.yaml:39]: Nit: extra space

File: serious_stuff/lib/screens/MapPage.dart (hunk 4 above)
Comment (jusung1459) [reply serious_stuff/lib/screens/MapPage.dart:7]: this constant is the center of sfu burnaby so well need it again, so we should put it somewhere

**Pull request merged**